hypothesis_bio/hypothesis_bio.py

```diff
@@ -241,43 +242,67 @@ def fasta(
     - `sequence_source`: The source of the sequence. Defaults to [`dna`](#dna).
     - `wrap_length`: The width to wrap the sequence on. If `None`, mixed sizes are used.
     - `allow_windows_line_endings`: Whether to allow `\\r\\n` in the linebreaks.
+    = `multiple_fasta`: Whether multiple FASTA sequences will be generated.
     """
     if comment_source is None:
         comment_source = text(alphabet=characters(min_codepoint=32, max_codepoint=126))
     if sequence_source is None:
         sequence_source = dna()
 
-    comment = draw(comment_source)
-    sequence = draw(sequence_source)
+    if num_fasta is None:
+        # randomly determine how many FASTA sequences to generate
+        num_fasta = draw(integers(min_value=0)) ##replace with numpy if needed
+
+    # Lists to hold comments and sequences for each FASTA sequence
```

Comment:
comment not needed

Reply:
Do you mean the comment in the line above or next to num_fasta?

Reply:
line 256